### PR TITLE
another solution for problem #13

### PR DESCRIPTION
**Pull request by rkalra247**

### CODE REVIEW

File: test/13.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ let solution = require('../solutions/13').solution;
 // solution = require('./yourSolution').solution;
 
 describe('reverse String', () => {
-  it('should return true if input string is a palindrome', () => {
-    expect(solution("racecar")).to.equal(true);
-  });
-  it('should return true if input string is a palindrome', () => {
-    expect(solution("palindrome")).to.equal(false);
-  });
+   it.only('should return true if input string is a palindrome', () => {
```
Comment (songz) [inline test/13.js]: remove `.only`

File: solutions/13.js [hunk 2]
```diff
@@ -1,3 +1,18 @@
+//Rahul Kalra
+//Checking to see if the string is palindrome
+
+const solution = (string) =>{
```
Comment (songz) [inline solutions/13.js]: please don't use data types as variable names (like string, array, etc). try `palindromeInput`?

File: solutions/13.js [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+  let reverse = "";
+  for(let i = string.length - 1; i >= 0; i--){
+    reverse += string [i];
+  }
+  if(reverse === string){
```
Comment (songz) [inline solutions/13.js]: `reverse === string` is a boolean. you can return it directly

**Comment by yjlim5** [comment]
Closing this because you have merge conflicts that needs to be resolved.

**Pull request closed without merging**